src/pages/Workspace/BcvPicker.jsx

```diff
@@ -86,6 +102,30 @@ function BcvPicker() {
                 )
             }
         </Menu>
+        <Dialog
+                open={isOpen}
+                onClose={handleClose}
+                slotProps={{
+                    paper: {
+                        component: 'form',
+                    },
+                }}
+            >
+                <DialogTitle><b>{doI18n("components:core-local-workspace:change_book", i18nRef.current)}</b></DialogTitle>
```

Comment:
Here and elsewhere, it's pages:core... not components:core... None of the i18n works for me.